src/main.c

```diff
@@ -70,1649 +70,1673 @@
-    float last_touch_time;
-    int mem_view_address;
-    int mem_dump_size;
-    int mem_dump_start_address;
-    bool sidebar_open;
-    se_keybind_state_t key;
-    se_controller_state_t controller;
-    se_game_info_t recently_loaded_games[SE_NUM_RECENT_PATHS];
-    int sorted_recently_loaded_games[SE_NUM_RECENT_PATHS];
-    int recent_games_sort_type;
-    persistent_settings_t settings;
-    persistent_settings_t last_saved_settings;
-    bool overlay_open;
-    se_emulator_stats_t emu_stats; 
-    // Utilize a watchdog channel to detect if the audio context has encountered an error
-    // and restart it if a problem occurred. 
-    int audio_watchdog_timer; 
-    int audio_watchdog_triggered; 
-    bool block_touchscreen;
-    bool test_runner_mode;
-    sg_shader lcd_prog;
-    sg_buffer quad_vb;
-    sg_pipeline lcd_pipeline;
-    se_file_browser_state_t file_browser; 
-    float mouse_pos[2];
-    bool mouse_button[3];
-    float menubar_hide_timer;
-    se_touch_controls_t touch_controls; 
-    se_search_paths_t paths;
-    se_search_paths_t last_saved_paths;
-    se_bios_info_t bios_info;
-    sg_image font_atlas_image;
-    uint8_t font_cache_page_valid[(SE_MAX_UNICODE_CODE_POINT+1)/SE_FONT_CACHE_PAGE_SIZE];
-    bool update_font_atlas;
-    sb_joy_t hcs_joypad; 
-    int editing_cheat_index; //-1 when not editing a cheat
-    char cheat_path[SB_FILE_PATH_SIZE];
-    ImFont* mono_font; 
-
-    uint32_t current_click_region_id;
-    uint32_t max_click_region_id;
-    uint32_t ui_type; 
-    bool fake_paths; 
-    char loaded_theme_path[SB_FILE_PATH_SIZE];
-    char loaded_custom_font_path[SB_FILE_PATH_SIZE];
-    se_custom_theme_t theme;
-    bool ran_from_launcher;
-    char search_buffer[32];
+  uint64_t       laptime;
+  sg_pass_action pass_action;
+  sg_image       image_stack[GUI_MAX_IMAGES_PER_FRAME];
+  int            current_image;
+  int            screen_width;
+  int            screen_height;
+  float          dpi_override;
+  int            button_state[SAPP_MAX_KEYCODES];
+  struct {
+    bool  active;
+    float pos[2];
+  } touch_points[SAPP_MAX_TOUCHPOINTS];
+  float                 last_touch_time;
+  int                   mem_view_address;
+  int                   mem_dump_size;
+  int                   mem_dump_start_address;
+  bool                  sidebar_open;
+  se_keybind_state_t    key;
+  se_controller_state_t controller;
+  se_game_info_t        recently_loaded_games[SE_NUM_RECENT_PATHS];
+  int                   sorted_recently_loaded_games[SE_NUM_RECENT_PATHS];
+  int                   recent_games_sort_type;
+  persistent_settings_t settings;
+  persistent_settings_t last_saved_settings;
+  bool                  overlay_open;
+  se_emulator_stats_t   emu_stats;
+  // Utilize a watchdog channel to detect if the audio context has encountered an error
+  // and restart it if a problem occurred.
+  int                     audio_watchdog_timer;
+  int                     audio_watchdog_triggered;
```

Comment:
Spacing here is a bit long

Reply:
The spacing is long in order to have alignment, we could add spaces to alleviate that

Reply:
things like this may happen a bit everywhere though so declarations are aligned, it is sadly impossible as far as I can see to give it a sort of threshold.